drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -1236,6 +1238,13 @@ static void usbd_work_handler(struct k_work *item)
 
 	while ((ev = usbd_evt_get()) != NULL) {
 
```

Comment:
Unrelated, but looks a bit weird to me to have a blank line as the first thing in a block.